### PR TITLE
Hook up API status endpoint

### PR DESCRIPTION
**Pull request by olafveerman**

### CODE REVIEW

File: app/assets/scripts/components/status-api.js [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
   displayName: 'StatusApi',
 
+  getInitialState: function () {
+    return {
+      health: null
+    }
+  },
+
+  componentDidMount: function () {
+    fetch(`${config.api}status`)
```
Comment (danielfdsilva) [inline app/assets/scripts/components/status-api.js]: I prefer to have no trailing slash in the config so you can do `${config.api}/status`. But it's more a cosmetic thing.

Comment (danielfdsilva) [inline app/assets/scripts/components/status-api.js]: I'll leave it at your discretion

**Comment by danielfdsilva** [comment]
Good to go!

File: package.json [hunk 2]
```diff
@@ -76,7 +76,9 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.9.1",
+    "lodash": "^4.16.5",
```
Comment (olafveerman) [inline package.json]: guess we don't need this

**Pull request merged**